doc/cephfs/cache-configuration.rst

```diff
@@ -237,8 +237,8 @@ all cached caps (the actual recall value is ``num_caps -
 mds_min_caps_per_client(100)``).
 
 Under certain circumstances, many "recall caps" requests can be sent so quickly
-that the "mon warning limit" exceeded, and the "clients failing to respond to
-cache pressure" message can be triggered. If the client does not release the
+that the "mon warning limit" is exceeded, and the "clients failing to respond
+to cache pressure" message can be triggered. If the client does not release the
```

Comment:
```suggestion
that the health warning is generated: "clients failing to respond
to cache pressure". If the client does not release the
```

Reply:
I've committed this change. Thanks, @batrick.